config/features/signage/field.field.node.sign.field_sign_slides.yml

```
@@ -16,10 +17,13 @@ translatable: false
 default_value: {  }
 default_value_callback: ''
 settings:
-  handler: views
+  handler: unpublished
   handler_settings:
-    view:
-      view_name: slides_entity_reference
-      display_name: entity_reference_1
-      arguments: {  }
+    target_bundles:
+      slide: slide
+    sort:
+      field: _none
```

```suggestion
      field: title
```

@james917 can you update this to sort by title like the original widget had?

https://github.com/uiowa/uiowa/pull/9022/files#diff-1d661fd48264fe288de34d28670b3f7d4f313fc879d42128edb403f017ae34b7L24-L26